README.md

```diff
@@ -249,39 +255,54 @@ ForageSDK will expose the following function to collect the EBT card number:
     ): ForageApiResponse<String>
 ```
 
+#### Parameter definitions
+
+- `merchantAccount`: A unique seven digit numeric string that [FNS](https://docs.joinforage.app/docs/ebt-online-101#food-and-nutrition-service-fns) issues to authorized EBT merchants.
+- `bearerToken`: A [session token](https://docs.joinforage.app/reference/create-session-token).
+- `customerId`: A unique ID for the end customer making the payment. If you use your internal customer ID, then we recommend that you hash the value before sending it on the payload.
+
+#### Example
+
 This is an example of usage inside an ACC ViewModel:
 
 ```kotlin
     fun onSubmit() = viewModelScope.launch {
         _isLoading.value = true
-    
+
         val response = ForageSDK.tokenizeEBTCard(
             merchantAccount = merchantAccount,
             bearerToken = bearer,
             // NOTE: The following line is for testing purposes only and should not be used in production.
             // Please replace this line with a real hashed customer ID value.
             customerId = UUID.randomUUID().toString()
         )
-    
+
         when (response) {
             is ForageApiResponse.Success -> {
                 val adapter: JsonAdapter<PaymentMethod> = moshi.adapter(PaymentMethod::class.java)
-    
+
                 val result = adapter.fromJson(response.data)
-    
+
                 _paymentMethod.value = result
             }
             is ForageApiResponse.Failure -> {
                 _error.value = response.message
             }
         }
-    
+
         _isLoading.value = false
     }
 ```
```

Comment:
Can we add an optional step 3 here for "persisting the PaymentMethod ref in your wallet". lots of our customers will have wallet functionality in their apps so that users see saved payment methods on their next login. We should note that here as it is a missing piece of backend functionality.